test/database_service/test_db_experiment_data.py

```diff
@@ -1073,7 +1081,7 @@ def _job2_result():
             exp_data.data(0)["counts"], [copied.data(0)["counts"], copied.data(1)["counts"]]
         )
 
-    def _get_job_result(self, circ_count, has_metadata=False):
+    def _get_job_result(self, circ_count, label_from=0, no_metadata=False):
```

Comment:
Why change `has_metadata` to `no_metadata`? Also, this changes the default behavior of `_get_job_result()` to always return metadata? I guess this didn't affect other tests.

Reply:
`has_metadata` was only used by one test to test the behavior when a result has metadata versus when it does not have metadata. `label_from` adds metadata. I thought it was cleaner for `label_from` to work by default, unless you requested `no_metdata` than to require `label_from` and `has_metadata` for that case. I don't feel strongly though. The two tests I modified are the only ones in the module that care about labels or other metadata.